src/library-authoring/edit-block/LibraryBlock/LibraryBlock.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import { ensureConfig, getConfig } from '@edx/frontend-platform';
```

Comment:
Thanks for doing the work to make the Ramshackle-like editor work with the new authoring page. This will be really helpful for demoing the MFE.